semgrep-core/src/parsing/tree_sitter/Parse_php_tree_sitter.ml

```diff
@@ -155,7 +155,7 @@ let map_text (env : env) (xs : CST.text) =
   Common.map
     (fun x ->
       match x with
-      | `LT tok -> (* < *) token env tok
+      | `Tok_prec_n1_pat_524a507 tok -> (* < *) token env tok
```

Comment:
This is because the grammar now uses the pattern `/</` instead of just the string literal `'<'`.